docs/pages/careers/code-infra-engineer.md

```diff
@@ -0,0 +1,139 @@
+
+Learn more about MUI's products in this blog post: [An introduction to the MUI ecosystem](https://mui.com/blog/mui-product-comparison/).
+
+## The culture
+
+MUI is a fully remote company with a team that spans the globe.
+The majority of our work is asynchronous, and we rely on written communication to collaborate.
+We're radically transparent: nearly all of our work happens in public.
+Each contributor has the freedom to decide how and when they work, and that work is primarily self-directed: it's your responsibility to define and complete your own tasks in a timely manner.
+
+For additional details about the culture, you can check our [careers](https://mui.com/careers/) and [about](https://mui.com/about/) pages and also our [public Handbook](https://mui-org.notion.site/Handbook-f086d47e10794d5e839aef9dc67f324b).
+
+## Why we're hiring
+
+Over the time, MUI expanded the product lines by adding more products.
+We currently active work across 5+ different repositories, and need to main several documentations.
+
+We want to build a team that will take care of the infrastructure work that's needed to make the work across different repositories/docs experiences seamless.
+We want to set up common utilities and set up best practices that all teams across the company can use.
+
+Overall, both our open-source community and our premium products are growing fast.
+We need talented people to keep up with the demands of our exigent community!
+
+### Why this is interesting
+
+You would be able to shape the infrstructure of one of the most visited reposotires and documentation.
+The impact of your work will not improve the experience for the MUI team, but also for thousands of other open source contributors that contribute to our projects.
+
+Our products empower React developers to build awesome applications faster – we see millions of developers on MUI's docs every year, one million a month.
+
+## The role
+
+### Key responsibilities
+
+Depending on the day, you'll:
+
+- **Keep technical debt in check**. You will make sure we can keep shipping features at a reasonable pace.
+- **Find consensus**. In response to their needs, this role aligns the teams on "one way" of doing things and doing it well. You'll need to communicate clearly and openly. You'll be at service to the teams DX needs.
+- **Be a solver**. You will dig deep into arbitrarily complex problems and find an appropriate path forward. Some focus on a given area for long periods, others bounce from hotspot to hotspot as guided by organizational leadership.
+- **Own infrastructure tasks**, like managing dependencies, analytics, TypeScript modules, and hosting, so product teams can stay focused on their primary responsibilities.
+- **Develop new features** that will enhance the way the libraries are showcased.
+
+### Tasks and initiatives you might take on
```

Comment:
- Enhance security of our documentation websites
- Improve documentation performance